wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/404.scss

```diff
@@ -0,0 +1,101 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+@import 'wagtailadmin/scss/components/logo';
+
+body {
+    font-family: $font-sans;
+    color: $color-white;
+
+    .page404__bg {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background-color: $color-teal-darker;
+
+        .wagtail-logo-container__desktop {
+            position: fixed;
+            right: calc(50% + 130px);
+            top: calc(45% - 250px);
```

Comment:
This shouldn't need `fixed` positioning either. From what I understand `absolute` will do. Something like:

```scss
top: 50%;
left: 50%;
transform: translate(-50%, 50%);
```

You could also try centering with Flexbox. On the parent element, `display:flex; justify-content:center; align-items:center;`.

Reply:
Good point, `absolute` should work fine. I'll implement that on Monday.

I'd also prefer to use `transform`, however this causes complications with the logo animations (as these use `transform` tags too). For example, if I used `transform: translate(-130px, -250px);` it would get overridden by the `transform: rotate(4deg);` used for the nod animation. It's just easier to use `calc` here.

Also, the bird isn't supposed to be centered -- it's slightly offset to the left to make way for the text.

Reply:
My assumption is that the combination of the logo + text are centered, so the centering code wouldn't be directly on the bird. From there, it would be ok to relatively offset the bird to the left, and the text to the right if required, or just add a margin between the two.

For calc, it only works here because you're using hard-coded values. The point of `transform: translate(-50%, 50%);` is that there is no need to specify the size of the image / block, it will work regardless.